sentence_transformers/SentenceTransformer.py

```diff
@@ -719,6 +725,12 @@ def fit(self,
                     if not skip_scheduler:
                         scheduler.step()
 
+                    if log_steps>0 and training_steps%log_steps==log_steps-1 and log_callable is not None:
+                        try:
+                            log_callable(train_idx, training_steps, scheduler.get_last_lr(), loss_value.item())
```

Comment:
I had added `global_step` to better track across epochs
```suggestion
                            log_callable(train_idx, training_steps, global_step, scheduler.get_last_lr(), loss_value.item())
```